CHANGELOG.md

```diff
@@ -5,10 +5,9 @@ All notable changes to this application will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
 
-### Added
-- Vulnerability: Host Header Injection
-- sitemap.xml
-- Dockerfile and docker-compose.yml
-- Procfile for deploying to heroku
```

Comment:
We aren't creating a release right now, so let's just keep everything under Unreleased. Also can you add all the vulnerabilities to the list?

Reply:
Resolved.